tests/manage/z_cluster/test_coredump_check_for_ceph_daemon_crash.py

```diff
@@ -47,30 +59,42 @@ def finalizer():
 
         request.addfinalizer(finalizer)
 
-    @pytest.mark.parametrize(
-        argnames=["daemon_type"],
-        argvalues=[
-            pytest.param(*["mon"], marks=pytest.mark.polarion_id("OCS-2491")),
-            pytest.param(*["osd"], marks=pytest.mark.polarion_id("OCS-2492")),
-            pytest.param(*["mgr"], marks=pytest.mark.polarion_id("OCS-2493")),
-        ],
-    )
-    def test_coredump_check_for_ceph_daemon_crash(self, daemon_type):
+    def test_coredump_check_for_ceph_daemon_crash(self):
         """
         Verify coredumpctl list updated after killing daemon
 
         """
-        log.info(f"Get Node name where {daemon_type} pod running")
-        if daemon_type == "mon":
-            mon_pod_nodes = [get_pod_node(pod) for pod in get_mon_pods()]
-            node_obj = mon_pod_nodes[0]
-        elif daemon_type == "mgr":
-            mgr_pod_nodes = [get_pod_node(pod) for pod in get_mgr_pods()]
-            node_obj = mgr_pod_nodes[0]
-        elif daemon_type == "osd":
-            osd_pod_nodes = [get_pod_node(pod) for pod in get_osd_pods()]
-            node_obj = osd_pod_nodes[0]
-        node_name = node_obj.name
+        log.info("Get Node name where mon pod running")
+        mon_pod_nodes = [get_pod_node(pod) for pod in get_mon_pods()]
+        mon_pod_node_names = [node.name for node in mon_pod_nodes]
+
+        log.info("Get Node name where mgr pod running")
+        mgr_pod_nodes = [get_pod_node(pod) for pod in get_mgr_pods()]
+        mgr_pod_node_names = [node.name for node in mgr_pod_nodes]
+
+        log.info("Get Node name where osd pod running")
+        osd_pod_nodes = [get_pod_node(pod) for pod in get_osd_pods()]
+        osd_pod_node_names = [node.name for node in osd_pod_nodes]
+
+        node_mgr_mon_osd_names = set(mgr_pod_node_names).intersection(
+            osd_pod_node_names, mon_pod_node_names
+        )
+        node_mgr_osd_names = set(mgr_pod_node_names).intersection(osd_pod_node_names)
+        node_mgr_mon_names = set(mgr_pod_node_names).intersection(osd_pod_node_names)
```

Comment:
I think here it should be: `node_mgr_mon_names = set(mgr_pod_node_names).intersection(mon_pod_node_names)`

Reply:
correct! I will fix it